app/org/sagebionetworks/bridge/play/controllers/ScheduleController.java

```diff
@@ -67,12 +72,18 @@ public Result getSchedules() throws Exception {
         UserSession session = getAuthenticatedAndConsentedSession();
         StudyIdentifier studyId = session.getStudyIdentifier();
         ClientInfo clientInfo = getClientInfoFromUserAgentHeader();
+
+        ScheduleContext context = new ScheduleContext.Builder()
+                .withUser(session.getUser()).withClientInfo(clientInfo).build();
         
         List<SchedulePlan> plans = schedulePlanService.getSchedulePlans(clientInfo, studyId);
+
         List<Schedule> schedules = Lists.newArrayListWithCapacity(plans.size());
         for (SchedulePlan plan : plans) {
-            Schedule schedule = plan.getStrategy().getScheduleForUser(session.getStudyIdentifier(), plan, session.getUser());
-            schedules.add(schedule);
+            Schedule schedule = plan.getStrategy().getScheduleForUser(plan, context);
+            if (schedule != null) {
+                schedules.add(schedule);
```

Comment:
Logging this. Added the user ID so we can log it along with the plan name, so we can trace back if need be.